### ISSUE TITLE
process.env is required

### ISSUE THREAD

**Issue by lexmes** [userlike/messenger#12]
Apparently the library assumes `process.env` is defined which it is not in browser environments. I use webpack do transpile my JS code but it skips all `node_modules` dependencies which results in `process.env` being undefined when the library is installed via npm and transpiled by webpack.

```
Uncaught (in promise) ReferenceError: process is not defined
    at getManifestUrl (webpack-internal:///../node_modules/@userlike/messenger-internal/dist/es/loader/load-scripts.js:131)
    at eval (webpack-internal:///../node_modules/@userlike/messenger-internal/dist/es/loader/load-scripts.js:89)
    at step (webpack-internal:///../node_modules/@userlike/messenger-internal/dist/es/loader/load-scripts.js:40)
    at Object.eval [as next] (webpack-internal:///../node_modules/@userlike/messenger-internal/dist/es/loader/load-scripts.js:21)
    at eval (webpack-internal:///../node_modules/@userlike/messenger-internal/dist/es/loader/load-scripts.js:15)
    at new Promise (<anonymous>)
    at __awaiter (webpack-internal:///../node_modules/@userlike/messenger-internal/dist/es/loader/load-scripts.js:11)
    at loadAssetsManifest (webpack-internal:///../node_modules/@userlike/messenger-internal/dist/es/loader/load-scripts.js:84)
    at loadScripts (webpack-internal:///../node_modules/@userlike/messenger-internal/dist/es/loader/load-scripts.js:51)
    at eval (webpack-internal:///../node_modules/@userlike/messenger/dist/es/index.js:60)
```

**Comment by anilanar** [comment]
@lexmes Are you using webpack 5?

**Comment by lexmes** [comment]
@anilanar Yep `5.0.0-beta.26`

**Comment by anilanar** [comment]
Thanks for the report. Will fix quickly :+1:

**Comment by lexmes** [comment]
Problem also exists with webpack `^5.20.0`

**Comment by anilanar** [comment]
@lexmes Fixed in `@userlike/messenger@1.1.1`.

**Comment by lexmes** [comment]
Thanks for the quick fix!
